app/controllers/chat_channel_memberships_controller.rb

```diff
@@ -24,15 +24,25 @@ def chat_channel_info
     @membership = ChatChannelMembership.find(params[:id])
     authorize @membership
     @channel = @membership.chat_channel
-    data = ChatChannelDetailPresenter.new(@channel, @membership).as_json
-
-    render json: { success: true, result: data, message: "" }, success: :ok
+    invitation_link = @channel.chat_channel_invitation_links.last
```

Comment:
@rhymes  I think we only need the last record. The reason behind it is always the last record that was `active` or `expired`. If we call all records then we need to find the active or expire record.

Reply:
Sorry, I meant the name of the relation, to be called `invitation_links` instead of `chat_channel_invitation_links` as `@chat_channel.` already qualifies them